packages/Filters/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@paprika/filters",
+  "version": "0.1.0",
+  "description": "Filters",
+  "author": "@paprika",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/Filters"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/button": "^1.0.4-alpha.0",
+    "@paprika/constants": "^1.0.0",
+    "@paprika/date-picker": "^1.0.5-alpha.0",
+    "@paprika/icon": "^1.0.1",
+    "@paprika/input": "^1.0.4-alpha.0",
+    "@paprika/panel": "^2.0.3-alpha.0",
+    "@paprika/radio": "^1.0.3-alpha.0",
+    "@paprika/select": "^1.0.3-alpha.0",
```

Comment:
Do you need `alpha` versions?

Reply:
Yes, I'll need to keep them in sync with the latest packages, so that lerna can take care of the rest